actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -24,6 +24,9 @@ class Railtie < Rails::Railtie # :nodoc:
       'X-Content-Type-Options' => 'nosniff'
     }
 
+    config.public_file_server = ActiveSupport::OrderedOptions.new
+    config.public_file_server.headers = { 'Cache-Control' => 'public, max-age=3600' }
```

Comment:
I'm not sure if this is the right place to define `public_file_server`. I'm open to suggestions.